tasks/uk/census/ons.py

```diff
@@ -0,0 +1,125 @@
+# http://webarchive.nationalarchives.gov.uk/20160105160709/http://ons.gov.uk/ons/guide-method/census/2011/census-data/bulk-data/bulk-data-downloads/index.html
+
+import csv
+import json
+import os
+import re
+import urllib
+import shutil
+from zipfile import ZipFile
+
+from luigi import Task, Parameter, LocalTarget
+
+from tasks.meta import current_session
+from tasks.util import classpath, shell, DownloadUnzipTask, TempTableTask
+
+
+class DownloadUK(Task):
+    API_URL = 'https://www.nomisweb.co.uk/api/v01/dataset/def.sdmx.json?search={}*'
+    DOWNLOAD_URL = 'https://www.nomisweb.co.uk/api/v01/dataset/{id}.bulk.csv?time=2011&measures=20100&geography={geo}'
+
+    table = Parameter()
+
+    def run(self):
+        # Query API, extract table ID from name
+        meta = json.load(urllib.urlopen(self.API_URL.format(self.table)))
```

Comment:
:_) at last using a library to make request. Why don't use requests?

Reply:
Why? In this case, this is very simple with the standard lib, so no need to pull an extra dependency. Of course, if we need to do something more complex (headers, cookies, ...) we could add it. But for this, I think `urllib` is fine.

Reply:
If we're going to add that dependency, which I'm pretty sure we're going to do because we have complex requests, why don't start using it here?